Sources/SpeziNotifications/Actions/RequestNotificationAuthorization.swift

```diff
@@ -19,8 +19,9 @@ extension Spezi {
         fileprivate init() {}
 
         /// Request notification authorization.
-        /// - Parameter options: The authorization options your app is requesting.
-        public func callAsFunction(options: UNAuthorizationOptions) async throws {
+        /// - parameter options: The authorization options your app is requesting.
+        /// - returns: A Boolean value indicating whether the user granted the requested permissions.
+        public func callAsFunction(options: UNAuthorizationOptions) async throws -> Bool {
```

Comment:
To avoid making this a breaking change, can we make this a discardable result?

Reply:
it's already a non-breaking change, since not using the non-discardable result is only a compiler warning but not an error.

IMO this is actually what we want, since this way someone using the function will be informed about the fact that they might want to do smth with the return value here. additionally, we also want future callers to get informed about using the return value